### DISCUSSION TITLE
Cant find how to "toPEM" after upgrade

### DISCUSSION THREAD

**Discussion by citkane** [panva/jose#418] [Q&A] [answered]
I have been successfully using jose [1.26.1] to generate .pem file as described in [this previous dicussion](https://github.com/panva/jose/discussions/39#discussioncomment-210336).

After upgrading to [4.8.1], I can't find a way to reproduce this functionality.
I replace: 
`const privateKey = jose.JWK.generateSync('RSA', 2048);`
with:
`const { publicKey, privateKey }= await generateKeyPair('RSA-OAEP', 2048);`
in order to get a keypair,
but then:
`privateKey.toPEM(true)`
yields the error:
`TypeError: privateKey.toPEM is not a function`

Has `toPem` been deprecated?
Maybe I am missing something obvious, but I can't find how to do this in the documentation.

Help appreciated please.

**Comment by panva** [comment, marked as answer]
There are individual [key export functions](https://github.com/panva/jose/tree/main/docs#:~:text=Symmetric%20Secret%20Generation-,Key%20Export,-JWK%20Export).
